app/pages/nonsipp/sharesdisposal/HowManyDisposalSharesPage.scala

```diff
@@ -22,7 +22,7 @@ import pages.QuestionPage
 import play.api.libs.json.JsPath
 import utils.RefinedUtils.RefinedIntOps
 
-case class HowManySharesPage(srn: Srn, index: Max5000, disposalIndex: Max50) extends QuestionPage[Int] {
+case class HowManyDisposalSharesPage(srn: Srn, index: Max5000, disposalIndex: Max50) extends QuestionPage[Int] {
```

Comment:
@ccatosdevelopment As you are working on this are in parallel, would this refactoring cause any problems to you?